lib/otel/segments/consumer.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+module.exports = createConsumerSegment
+
+// Notes:
+// + https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/messaging/messaging-spans.md
+// + We probably want to inspect `messaging.system` so that we can generate
+// attributes according to our own internal specs.
+
+const Transaction = require('../../transaction/')
+const { DESTINATIONS, TYPES } = Transaction
+
+const {
+  // SEMATTRS_MESSAGING_SYSTEM,
```

Comment:
the attributes you'll need are the same as the producer, [see](https://github.com/newrelic/node-newrelic/pull/2839/files#diff-60300fb5dc247f4d27b053386d7e3a4964226954788a73c6d36effb49b777f18R7)